### PR TITLE
chore: Moved over components from chaotictempest.near account

### PR DESCRIPTION
**Pull request by ChaoticTempest**
Found some free time, so moved over the components that was still left over in the chaotictempest account. @milenakrawczyk this should resolve all the issues on your side

### CODE REVIEW

File: src/Search/FullPage/AccountProfileCard.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+const TagsWrapper = styled.div`
+  padding-top: 4px;
+`;
+
+return (
+  <Card>
+    <CardLeft>
+      <Avatar href={profileUrl} onPointerUp={onPointerUp}>
+        <Widget
+          src="mob.near/widget/Image"
```
Comment (milenakrawczyk) [inline src/Search/FullPage/AccountProfileCard.jsx:96]: This should be changed to "${REPL_MOB}/widget/Image".

**Comment by shelegdmitriy** [comment]
@ChaoticTempest would you mind to resolve comments so we could merge it? It seems to be a really quick fix

File: src/Search/FullPage/AccountProfileCard.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
+const accountId = props.accountId;
+const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
+const tags = Object.keys(profile.tags || {});
+const profileUrl = `/near/widget/ProfilePage?accountId=${accountId}`;
```
[the remark this answers is not in the document]
Comment (ChaoticTempest) [reply src/Search/FullPage/AccountProfileCard.jsx]: ahh, I made those changes locally but forgot to push it

**Review by milenakrawczyk — APPROVED** [review]
LGTM!

**Pull request merged**